src/dashboards.js

```diff
@@ -388,3 +512,17 @@ function expandOSs(OSs) {
   });
   return osVersions;
 }
+
+function updateOSs() {
+  // Update CSS classes for labels marking whether they are all selected
+  var allSelectedOSList = compressOSs().filter(function(os) { return os.indexOf(",") < 0; }); // List of all OSs that are all selected
+  var selector = $("#filter-os").next().find(".multiselect-container");
+  selector.find(".multiselect-group-clickable").removeClass("all-selected");
+  var optionsMap = {};
+  getHumanReadableOptions("os", allSelectedOSList).forEach(function(option) { optionsMap[option[0]] = option[1]; });
```

Comment:
you're really packing in the statements into lines, use the whitespace, it's free :)